install/builder.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
+
+import (
+	"context"
+	"github.com/kiegroup/container-builder/client"
+	v08 "github.com/kiegroup/kogito-serverless-operator/api/v1alpha08"
+	"github.com/kiegroup/kogito-serverless-operator/utils/resources"
+)
+
+func installBuilderServiceAccountRolesOpenShift(ctx context.Context, c client.Client, namespace string) error {
+	return resources.ResourcesOrCollect(ctx, c, namespace, nil, true, resources.IdentityResourceCustomizer,
+		"/resources/builder/builder-service-account.yaml",
+		"/resources/builder/builder-role.yaml",
+		"/resources/builder/builder-role-binding.yaml",
+		"/resources/builder/builder-role-openshift.yaml",
+		"/resources/builder/builder-role-binding-openshift.yaml",
+	)
+}
+
+func installBuilderServiceAccountRolesKubernetes(ctx context.Context, c client.Client, namespace string) error {
```

Comment:
Can't we use OLM to install this for us? If not available, we then install it manually.

Reply:
I think this one will depend on what would be the builder namespace or ?

Reply:
Yeah, I think we can rely on the `kn workflow` CLI to do this task. Secretly creating roles might not be a good option to go.

Reply:
If this is something that should not be hidden, and will be integrated into the `kn workflow` CLI, IMO, we should:

- remove that part from the operator code
- setup a manual command into the documentation to setup this by the user